File: Frontend/voitheia-App/src/app/common/models/updateUserDTO.ts
import { MinimalUserDTO } from './minimalUserDTO';

export class UpdateUserDTO extends MinimalUserDTO {
    lastName: string;
    street: string;
    zip: string;
    city: string;
    country: string;

    constructor() {
        super();

        this.lastName = "";
        this.street = "";
        this.zip = "";
        this.city = "";
        this.country= "Deutschland";
    }
}

File: Backend/ActiveCruzer/Models/DTO/UpdateUserDto.cs
using System.ComponentModel.DataAnnotations;

namespace ActiveCruzer.Models.DTO
{
    public class UpdateUserDto
    {
        [Required]
        public string FirstName { get; set; }
        [Required]
        public string LastName { get; set; }
        [Required]
        public string Street { get; set; }
        public string Zip { get; set; }
        [Required]
        public string City { get; set; }
        [Required]
        public string Country { get; set; }
    }
}